plugins/UsersManager/API.php

```diff
@@ -780,7 +819,23 @@ private function isUserTheOnlyUserHavingSuperUserAccess($userLogin)
     }
 
     /**
-     * Generates a unique MD5 for the given login & password
+     * Generates a new random authentication token.
+     *
+     * @param string $userLogin Login
+     * @return string
+     */
+    public function createTokenAuth($userLogin)
+    {
+        return hash(
```

Comment:
Is there an advantage to making the API token sha256 over md5? one downside which I'm only thinking about now is that changing the `token_auth` length also changes our APIs which may need some changes in SDK APIs, as SDKs tend to test that token_auth is 32 chars or throw an exception otherwise (as we do this in the php sdk). Maybe after all it would make sense to keep md5 for api tokens?